_inc/client/setup-wizard/feature-toggle/map-feature-to-props.js

```diff
@@ -293,6 +295,39 @@ const features = {
 		},
 	},
 
+	'creative-mail': {
+		mapStateToProps: state => {
+			const isCreativeMailActive = isPluginActive(
+				state,
+				'creative-mail-by-constant-contact/creative-mail-plugin.php'
+			);
+
+			return {
+				feature: 'creative-mail',
+				title: __( 'Creative Mail by Constant Contact', 'jetpack' ),
+				details: __( 'Turn visitors into subscribers with email marketing.', 'jetpack' ),
+				checked: isCreativeMailActive,
+				isDisabled: true,
+				isPaid: true,
+				configureLink: isCreativeMailActive ? '/wp-admin/admin.php?page=creativemail' : null,
+				learnMoreLink:
+					'https://jetpack.com/support/jetpack-blocks/subscriber-form-block-and-field/',
```

Comment:
I've updated this link to be a bit more descriptive.

```suggestion
					'https://jetpack.com/support/jetpack-blocks/form-block/newsletter-sign-up-form/',
```